editor/plugins/script_text_editor.cpp

```diff
@@ -609,6 +609,18 @@ void ScriptTextEditor::_validate_script() {
 	for (List<ScriptLanguage::Warning>::Element *E = warnings.front(); E; E = E->next()) {
 		ScriptLanguage::Warning w = E->get();
 
+		Dictionary ignore_meta;
+		ignore_meta["line"] = w.line;
+		ignore_meta["code"] = w.string_code.to_lower();
+		warnings_panel->push_cell();
+		warnings_panel->push_meta(ignore_meta);
+		warnings_panel->push_color(
+				warnings_panel->get_theme_color("accent_color", "Editor").lerp(warnings_panel->get_theme_color("mono_color", "Editor"), 0.5));
+		warnings_panel->add_text(TTR("[Ignore]"));
```

Comment:
I wonder if styling it as `[ Ignore ]` would give even more of a "I'm clickable" vibe?

Reply:
It looks like this with the additional spacing: 

![Script editor warning](https://user-images.githubusercontent.com/180032/83670576-aef20300-a5d3-11ea-9cf2-9266191458cb.png)

Compared to before:

![Script editor warning](https://user-images.githubusercontent.com/180032/82754143-4e8ce580-9dcb-11ea-9d3e-b9d23983dbcc.png)

Reply:
At least it's not something we're doing currently, so I guess let's go with what is current implemented. We can always refine later on.